orttraining/orttraining/python/training/ortmodule/_fallback.py

```diff
@@ -171,7 +172,7 @@ def fallback(self, log_level: _logger.LogLevel, *inputs, **kwargs):
 
         assert self.is_pending(), "`fallback` can only be called when there is a pending fallback"
 
-        if log_level <= _logger.LogLevel.WARNING:
+        if log_level <= _logger.LogLevel.WARNING and not self._raised_ortmodule_onnx_model_exception:
             warnings.warn(
```

Comment:
Please add/modify Unit Test to verify we only get fallback warning message one time

Reply:
Ok, sure, thank you. I will add unit test for this PR.